### PR TITLE
chore(suite): hide pass manager navigation item

### PR DESCRIPTION
**Pull request by MiroslavProchazka**
https://github.com/user-attachments/assets/dea87a6a-a825-4561-948c-472e8ef175ae

### CODE REVIEW

File: packages/suite/src/constants/suite/experimental.ts [hunk 1]
```diff
@@ -19,6 +21,8 @@ export const EXPERIMENTAL_FEATURES: Record<ExperimentalFeature, ExperimentalFeat
         title: 'TR_EXPERIMENTAL_PASSWORD_MANAGER',
         description: 'TR_EXPERIMENTAL_PASSWORD_MANAGER_DESCRIPTION',
         knowledgeBaseUrl: EXPERIMENTAL_PASSWORD_MANAGER_KB_URL,
+        goToRoute: 'password-manager',
+        routes: ['password-manager'],
```
Comment (Lemonexe) [inline packages/suite/src/constants/suite/experimental.ts]: `routes` is unused

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ import { EXPERIMENTAL_FEATURES, ExperimentalFeature } from 'src/constants/suite/
 import { useDispatch, useSelector } from 'src/hooks/suite';
 import { selectIsDebugModeActive } from 'src/reducers/suite/suiteReducer';
 
+import { Link } from 'react-router-dom';
```
Comment (Lemonexe) [inline packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: This gives lint error (imports sorting)
:point_right: run `yarn lint:js --fix` to automatically sort them as per eslint :slightly_smiling_face:

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx [hunk 3]
```diff
@@ -59,7 +61,15 @@ const FeatureLine = ({ feature, enabledFeatures }: FeatureLineProps) => {
                 buttonTitle={<Translation id="TR_LEARN_MORE" />}
             />
             <ActionColumn>
-                <Checkbox isChecked={checked} onClick={onChangeFeature} />
+                {config.goToRoute ? (
+                    <Link to={config.goToRoute}>
```
Comment (Lemonexe) [inline packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: :thinking:  Curiously, we never used `<Link to` so far, even though it's the simplest way to work with react-router.
Instead, we use own function [goto](https://github.com/trezor/trezor-suite/blob/3f257bcad4e2620c6c33af7450a2cc640e4dd940/packages/suite/src/actions/suite/routerActions.ts#L99), which has more complex logic.
But this seems sufficient to me, works as expected.

**Review by Lemonexe — APPROVED** [review]
Looks great :+1: 
:memo: some comments to be addresed

File: packages/suite/src/constants/suite/experimental.ts [hunk 4]
```diff
@@ -19,6 +21,8 @@ export const EXPERIMENTAL_FEATURES: Record<ExperimentalFeature, ExperimentalFeat
         title: 'TR_EXPERIMENTAL_PASSWORD_MANAGER',
         description: 'TR_EXPERIMENTAL_PASSWORD_MANAGER_DESCRIPTION',
         knowledgeBaseUrl: EXPERIMENTAL_PASSWORD_MANAGER_KB_URL,
+        goToRoute: 'password-manager',
```
Comment (jvaclavik) [inline packages/suite/src/constants/suite/experimental.ts]: Please rename `goToRoute` to `routeName`

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx (hunk 3 above)
Comment (jvaclavik) [reply packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: Please use it this way (without Link):

```

const handleClick = () => {
  goTo(config.routeName)
}

...

<Button variant="tertiary" onClick={handleClick}>
  <Translation id="TR_GO_TO_EXP_FEATURE" />
</Button
```

File: packages/suite/src/constants/suite/experimental.ts (hunk 1 above)
Comment (MiroslavProchazka) [reply packages/suite/src/constants/suite/experimental.ts]: Solved in 919e03efc1165779a51769ce3298867753cfe1ba

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx (hunk 2 above)
Comment (MiroslavProchazka) [reply packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: Solved in 919e03efc1165779a51769ce3298867753cfe1ba

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx (hunk 3 above)
Comment (MiroslavProchazka) [reply packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: Solved in 919e03efc1165779a51769ce3298867753cfe1ba

File: packages/suite/src/constants/suite/experimental.ts (hunk 4 above)
Comment (MiroslavProchazka) [reply packages/suite/src/constants/suite/experimental.ts]: Solved in 919e03efc1165779a51769ce3298867753cfe1ba

**Review by jvaclavik — APPROVED** [review]

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx [hunk 5]
```diff
@@ -49,6 +51,10 @@ const FeatureLine = ({ feature, enabledFeatures }: FeatureLineProps) => {
             },
         });
     };
+    const handleClick = () => {
+        if (!config.routeName) return;
+        dispatch(goto(config.routeName as Route['name']));
```
Comment (Lemonexe) [inline packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]:
```suggestion
        dispatch(goto(config.routeName));
```
And instead change  `routeName?: Route['name']` in `experimental.ts` :slightly_smiling_face:

**Review by Lemonexe — APPROVED** [review]
Almost there :muscle:

File: packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx (hunk 5 above)
Comment (MiroslavProchazka) [reply packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx]: Fixed in 1f2c2b5961e9da0af54d36f662b25619149744ba

**Review by Lemonexe — APPROVED** [review]

**Pull request merged**